### DISCUSSION TITLE
Can not work with ESP 2.0.11 library - old version (1.06?) better

### DISCUSSION THREAD

**Discussion by joopheuvel** [espressif/arduino-esp32#8633] [Q&A] [answered]
In IDE2 and with an Olimex-ESP32-EVB board, I updated the board library I was using until now to the latest version 2.0.11. After compile (btw memory usage of code (no spiffs) went from 79% to 88%), code did not work anymore. Getting connection failures on ethernet and wifi versions of the board (with firebase but also other communications). Wanted to go back to older version, the one I had 3 months ago (1.0.6 works fine). Two questions:
1- are there still toothing issues with 2.0.11? should I wait, is my board incompatible?
2- do I need to stick with latest V1 version (V1.0.6)
Many thanks...
Joop
Netherlands

**Comment by mrengineer7777** [comment]
Possibly related https://github.com/espressif/arduino-esp32/issues/8482.  Should be fixed in 2.0.12.

**Reply by joopheuvel** [reply, marked as answer]
Thanks!
I have some more info. deciced to do a clean remove, reinstall of the latest arduino IDE, load all libraries and board (Olimex-ESP32-EVB) and give it one more try. It compiles properly now with 2.0.11. It still uses much more on the memory resource compared to 1.06 though - but should be okay. The problem that I am running into now is that after EXACTLY 100 firebase.rtdb operations (mix of read,write,int,string) it ALWAYS stops with a connection refused error. Seems to be a firabase issue.

21:32:42.590 -> Loop, 100
21:32:43.573 -> Status, Heap:163k, FBFail:0, FBRe:0, FBSucc:100, Idiscon:0, Loops:16, FBbusy:4s
21:32:43.618 -> FB ready
21:32:43.618 -> FBReadIntStart: /Home/stations/GM10_Out/04_station_attention
21:33:43.665 -> After 60070ms retry_1_ReadInt /Home/stations/GM10_Out/04_station_attention connection refused!

The timeout is exactly after 60 seconds and always after operation number 100, independent of delays, timings, other stuff going on. Cannot figure out how to tackle this one. Couldn't find anything online. Will also ty the firebase forum

Thanks!
Joop
